apps/desktop/src/lib/vbranches/types.ts

```diff
@@ -188,10 +188,7 @@ export class DetailedCommit {
 
 	get status(): CommitStatus {
 		if (this.isIntegrated) return 'integrated';
-		if (
-			(this.isRemote && (!this.relatedTo || this.id === this.relatedTo.id)) ||
-			(this.copiedFromRemoteId && this.relatedTo && this.copiedFromRemoteId === this.relatedTo.id)
-		)
+		if (this.isRemote && (!this.relatedTo || this.id === this.relatedTo.id))
```

Comment:
Wonderful, less really is more!

I also tried my use-case again, which is to create a vbranch from another branch that has no change-ids in commits, and then undid one commit, and it also just works.

<img width="367" alt="Screenshot 2024-09-01 at 18 57 19" src="https://github.com/user-attachments/assets/ab33d784-9b5d-49a4-9200-c3af62077dcc">

I think what I wanted to accomplish is to have it show as `Local and remote` just as if change-ids were equal (or it's the same actual commit), but probably I misunderstood the state (or status) that such a copied commit is actually in.

And somewhat magically, it is all working even when the commit message is amended to the local commit.